flags/eigendaflags/cli.go

```diff
@@ -8,28 +8,46 @@ import (
 	"github.com/Layr-Labs/eigenda-proxy/common/consts"
 	"github.com/Layr-Labs/eigenda/api/clients"
 	"github.com/Layr-Labs/eigenda/api/clients/codecs"
+	v2_clients "github.com/Layr-Labs/eigenda/api/clients/v2"
 	"github.com/urfave/cli/v2"
 )
 
-// TODO: we should eventually move all of these flags into the eigenda repo
-
 var (
-	DisperserRPCFlagName                 = withFlagPrefix("disperser-rpc")
-	ResponseTimeoutFlagName              = withFlagPrefix("response-timeout")
-	ConfirmationTimeoutFlagName          = withFlagPrefix("confirmation-timeout")
-	StatusQueryRetryIntervalFlagName     = withFlagPrefix("status-query-retry-interval")
-	StatusQueryTimeoutFlagName           = withFlagPrefix("status-query-timeout")
-	DisableTLSFlagName                   = withFlagPrefix("disable-tls")
-	CustomQuorumIDsFlagName              = withFlagPrefix("custom-quorum-ids")
-	SignerPrivateKeyHexFlagName          = withFlagPrefix("signer-private-key-hex")
-	PutBlobEncodingVersionFlagName       = withFlagPrefix("put-blob-encoding-version")
+	// This is a temporary feature flag that will be deprecated once all client
+	// dependencies migrate to using EigenDA V2 network
+	V2Enabled = withFlagPrefix("v2-enabled")
+
+	// disperser specific flags (interoperable && mutex for (v1 && v2))
```

Comment:
what does "interoperable && mutex" mean?

Reply:
I think v1 and v2 should have completely independent flags. possibly one might want to set a different confirmation timeout for v1 and v2 (which makes sense given that there's no bridging on v2 for eg)

Reply:
But if we use a different flag namespace (e.g, `v2`) then we'd have to kill that namespace eventually in the future when we migrate to V2 which would cause eventual deprecation. Nbd but something to take into account

Reply:
https://github.com/Layr-Labs/eigenda-proxy/pull/296